spec.md

```diff
@@ -2330,6 +2339,15 @@ message NodeExpandVolumeRequest {
   // plugin MAY expand the volume to its maximum capacity.
   // This field is OPTIONAL.
   CapacityRange capacity_range = 3;
+
+  // Volume capability describing how the CO intends to use this volume.
+  // This allows SP to determine if volume is being used as a block
+  // device or mounted file system. For example - if volume is being
+  // used as a block device the SP MAY choose to skip expanding the
+  // filesystem. If volume_capability is omitted the SP MAY determine
+  // access_type from given volume_path for the volume and perform
```

Comment:
>If volume_capability is omitted the SP MAY determine
// access_type from given volume_path for the volume and perform
// node expansion.

I know this is `MAY` but it feels a little bit prescriptive. Do we want to give this suggestion if this is not something a `SP` "should" be doing? I feel like it's a specific workaround that may not always be possible to do from the `volume_path`

Reply:
can you give me some examples where the it may not be possible to determine access_type from given volume_path?

Reply:
>  Do we want to give this suggestion if this is not something a SP "should" be doing? 

I think the only way to convey that is to convert this field from OPTIONAL to REQUIRED and MAY to MUST.

But then we will have to introduce a new node capability for being backward compatible.

Reply:
> can you give me some examples where it may not be possible to determine access_type from given volume_path

I don't have any :) Ack on the backward compatibility, think thats ok